qiime_studio/server.py

```diff
@@ -13,6 +13,14 @@
 studio.after_request(add_cors_headers)
 
 
+# No output is visible, as for some reason it is held back
+# by the server flushed at seemingly random intervals
+# also, what should be the stdout, is being sent through stderr??
+def flush_stdio():
+    sys.stdout.flush()
+    sys.stderr.flush()
+
```

Comment:
It's really weird that this has to be done, but I guess the `WSGIServer()` doesn't output in a normal fashion. For one, every single line of output is actually pushed to `sys.stderr`, and without flushing on every request, it will not output anything to the console other than just randomly dropping about 15 lines of output (most consistently when a job is executed).